modules/qt/hm.nix

```diff
@@ -86,11 +94,19 @@
             ''
               [Appearance]
             ''
-            + lib.optionalString (config.qt.style ? name) ''
+            + lib.optionalString (config.qt.style.name != null) ''
               style=${config.qt.style.name}
             ''
             + lib.optionalString (iconTheme != null) ''
               icon_theme=${iconTheme}
+            ''
+            + lib.optionalString config.stylix.targets.qt.usePortal ''
+              standard_dialogs=xdgdesktopportal
+            ''
```

Comment:
Elaborate on this, as it should normally not need to be set.

Reply:
As I said in the main post, from my own usage portals seem to follow theming better.

They follow the theming of the desktop portal the user has installed (as in GTK desktop portal which seems to be the most used, will follow GTK theme). They can bring a more uniform look, since GTK applications already by default use them.

But it sounds like this is not wished to be added(?) so I'll remove it so this PR will contain just the font changes

Reply:
This would be a good reason to make this setting a default. However, this decision is up to @danth